### ISSUE TITLE
setup test pipeline on abn

### ISSUE THREAD

**Issue by matteodelucchi** [furrer-lab/abn#5]
move testpipline from private devel-abn repo to this public repo.

**Comment by j-i-l** [comment]
Addressing this issue makes sense only once we have the testing pipeline set up on devel

**Comment by j-i-l** [comment]
Duplicate of #25

**Comment by j-i-l** [comment]
Duplicate of #25
